### PR TITLE
ws and rest indices

### PR DESCRIPTION
**Pull request by antdjohns**

### CODE REVIEW

File: polygon/rest/models/snapshot.py [hunk 1]
```diff
@@ -31,6 +31,52 @@ def from_dict(d):
         )
 
 
+@modelclass
+class DayOptionContractSnapshot:
```
Comment (aitzkovitz) [inline polygon/rest/models/snapshot.py:35]: the indices snapshot endpoint only returns open, high, low, close, change, change_percent, and previous_close. so we shouldn't need volume, VWAP, and last_updated

**Comment by aitzkovitz** [comment]
looks good, you'll also need to update the UnderlyingAsset class in rest/models/snapshot.py to include a `value` field for index option snapshots.

File: polygon/rest/models/snapshot.py (hunk 1 above)
Comment (antdjohns) [reply polygon/rest/models/snapshot.py:35]: im reusing the class, I just moved it up. its none, shouldn't return right?

Comment (aitzkovitz) [reply polygon/rest/models/snapshot.py:35]: yeah it should be fine if you're reusing then, nvmd

**Review by chaig15 — APPROVED** [review]

**Comment by chaig15** [comment]
lgtm

**Review by aitzkovitz — APPROVED** [review]

File: polygon/rest/models/snapshot.py (hunk 1 above)
Comment (jbonzo) [reply polygon/rest/models/snapshot.py:35]: If this isn't being re-used can you move it back where it was to get it out of the diff

**Review by jbonzo — APPROVED** [review]
Just one thing but looks good

**Comment by antdjohns** [comment]
> Just one thing but looks good

what's the one thing? am I good to merge?

File: polygon/rest/models/snapshot.py (hunk 1 above)
Comment (jbonzo) [reply polygon/rest/models/snapshot.py]: @antdjohns This is the one thing.

**Pull request merged**